packages/tools/client-debugger/devtools/package.json

```diff
@@ -0,0 +1,96 @@
+{
+	"name": "@fluid-experimental/devtools",
+	"version": "2.0.0-internal.4.2.0",
+	"private": true,
+	"description": "Fluid Framework developer tools",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/tools/client-debugger/devtools"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "npm run build:genver && concurrently npm:build:compile npm:lint && npm run build:docs",
+		"build:compile": "npm run build:tsc",
+		"build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"build:full": "npm run build",
+		"build:full:compile": "npm run build:compile",
+		"build:genver": "gen-version",
+		"build:tsc": "npm run tsc",
+		"ci:build:docs": "api-extractor run && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../../_api-extractor-temp/",
+		"clean": "rimraf _api-extractor-temp nyc dist *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"format": "npm run prettier:fix",
+		"format-and-build": "npm run format && npm run build",
+		"format-and-compile": "npm run format && npm run build:compile",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
+		"rebuild": "npm run clean && npm run build",
+		"test": "echo TODO: add tests",
+		"tsc": "tsc"
+	},
+	"nyc": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.ts",
+			"dist/test/**/*.js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.ts",
+			"dist/**/*.js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
+	"dependencies": {
+		"@fluid-experimental/devtools-core": "workspace:~",
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/common-utils": "^1.1.1",
+		"@fluidframework/container-utils": "workspace:~",
+		"@fluidframework/fluid-static": "workspace:~"
+	},
+	"devDependencies": {
+		"@fluidframework/build-common": "^1.1.0",
+		"@fluidframework/build-tools": "^0.17.0",
+		"@fluidframework/eslint-config-fluid": "^2.0.0",
+		"@fluidframework/mocha-test-setup": "workspace:~",
+		"@fluidframework/test-runtime-utils": "workspace:~",
+		"@microsoft/api-extractor": "^7.34.4",
+		"@types/chai": "^4.0.0",
+		"@types/mocha": "^9.1.1",
+		"chai": "^4.2.0",
+		"concurrently": "^7.6.0",
```

Comment:
Not necessary here, we're not using it.

Reply:
`concurrently`? We're still using it in the `build` script.

Reply:
Ah, I missed it because it wasn't at the beginning of the script haha, my bad.